src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/dashboard/health/health.component.scss

```diff
@@ -60,3 +60,7 @@ table.ceph-chartbox {
     }
   }
 }
+
+.log {
```

Comment:
Maybe ``.text-monospace`` or ``.monospace-text`` says more about the intention of this style.

Reply:
I agree, `.log` is somewhat unintuitive.

Reply:
CSS class was renamed to `text-monospace`, thanks for the suggestion.